[DOCS-10675] Update Heatmaps snapshot user experience in RUM and PA (#28966)
DataDog/documentation: content/en/real_user_monitoring/session_replay/heatmaps.md

## Getting started

Navigate to [**Digital Experience > Real User Monitoring > Session Replay > Heatmaps**][2]. Select your application and view.

On the [Real User Monitoring landing page][3], select your application from the [application selector][4] and view. To the left of the timeframe selector, you can select the type of heatmap you would like to view: Top Elements, Click Map, or Scroll Map. This takes you to the [heatmap page][2] for a particular view.

{{< img src="real_user_monitoring/session_replay/heatmaps/heatmaps-different-views.png" alt="The heatmaps page has multiple ways to show different views: by application, map type, device type, action name, and granular filters." style="width:100%;">}}

You have the following additional view options:

- To switch the view being shown, use the **View Name** and **Application** selectors at the top.
- To change the device view, use the **Device type** selector.
- To filter by action name, use the **Filter actions by** dropdown.
- To add more granular filters, like a specific geography for example, click the **Add Filter** button.

## Top Elements

> Top Elements aggregate click actions on a given view by displaying the most interacted-with elements and their rank. The ranking on the map itself corresponds to the action name on the side.

Reviewer: ```suggestion
Top Elements heatmaps aggregate click actions on a given view by displaying the elements that are most interacted with, in addition to their interaction rank. The ranking on the map itself corresponds to the action name on the side.
```

Copied the edit you made in Product Analytics for this section

{{< img src="real_user_monitoring/session_replay/heatmaps/heatmaps-top-elements.png" alt="A ranking of the top elements clicked on a page." style="width:100%;">}}

Hover over any action name in the panel to highlight the corresponding action on the map.

## Click maps

> A Click map shows you the most interacted-with actions on a given view by aggregating user click actions from sessions and visualizing them as blobs on the map.

Reviewer: ```suggestion
A Click map shows you the actions most interacted with on a given view by aggregating user click actions from sessions and visualizing them as blobs on the map.
```

Copied the section you edited in the Product analytics section

### The view that I selected is not showing the initial content.

Heatmaps are generated with Session Replay data. Datadog's intelligent algorithm picks a replay that is both recent and best matches the initial state of the page. In some cases, you might not be able to find the correct replay. To switch the snapshot of your heatmap, you can use the **Change Snapshot** button to navigate through the different states of a replay and find the one you are looking for. If the replay you're viewing does not have the snapshot you're looking for, you can use the **Choose Another Replay** button to select another replay of the same view.

{{< img src="real_user_monitoring/session_replay/heatmaps/heatmaps-change-the-snapshot.mp4" alt="Select a different background by clicking the Change Snapshot button" video=true >}}

> ### On the action list on the side of my heatmap, I see an icon showing an element that is not visibile in the heatmap.

Reviewer: ```suggestion
### On the action list on the side of my heatmap, I see an icon showing an element that is not visible in the heatmap.
```

> ### On the action list on the side of my heatmap, I see an icon showing an element that is not visibile in the heatmap.

Reviewer: ```suggestion
### On the action list on the side of my heatmap, I see an icon showing an element that is not visible in the heatmap.
```
